Makefile

```diff
@@ -17,6 +17,7 @@ hubble:
 	$(GO) build $(if $(GO_TAGS),-tags $(GO_TAGS)) -ldflags "-w -s -X 'github.com/cilium/hubble/pkg.GitBranch=${GIT_BRANCH}' -X 'github.com/cilium/hubble/pkg.GitHash=$(GIT_HASH)' -X 'github.com/cilium/hubble/pkg.Version=${VERSION}'" -o $(TARGET)
 
 release:
+	rm -rf release; \
```

Comment:
doesnt really have to be part of the same command through `;`, can be it's own line

Reply:
I'd suggest to `release: clean` instead as the `clean` target takes care of this already.